docker/grafana/Dockerfile

```diff
@@ -14,7 +14,8 @@ VOLUME /dashboards
 ENV GF_SECURITY_ADMIN_USER=admin
 ENV GF_SECURITY_ADMIN_PASSWORD=admin
 
-ENV PROMETHEUS_URL=http://host.docker.internal:9090
+# Modified datasource to work with a network_mode: host
```

Comment:
Would be good to update the comment here, we are not using `network_mode: host` anymore

Reply:
If we don't modify this image because it might be used outside, I'd rather leave this comment that explains this env variable in the first place.

Reply:
So do you suggest to remove the comment?

Reply:
We might not even have to modify the prometheus URL anymore becasue we dropped `network_mode: host`

Reply:
Others might rely on this behavior, I think it's out of scope of this PR. We discussed that before already.